client/src/styles/_bootstrap-variables.scss

```diff
@@ -235,14 +229,14 @@ $line-height-lg: (4 / 3);
 $line-height-sm: 1.5;
 
 //$border-width: 1px !default;
-$border-color: darken(#f6f7f8, 10);
+$border-color: lighten($gray-200, 2);
```

Comment:
same as above, is this different from the other `gray-*` colour codes?

Reply:
I can see it make sense do this on a color that is used on a large surface but with color to be used with border properly 2% is hardly any different.